tests/slow/src/test/scala/tests/sbt/SbtBloopLspSuite.scala

```diff
@@ -439,7 +439,8 @@ class SbtBloopLspSuite
     } yield ()
   }
 
-  test("definition-deps") {
+  // TODO latest bloop-sbt plugin does not properly export sources
```

Comment:
I am looking currently at the sbt-bloop issue. I will create anyway an issue to track this.

Reply:
Sounds good. I have create this in sbt as well https://github.com/sbt/sbt/issues/6203